src/actions/ui.js

```diff
@@ -155,14 +155,17 @@ export function clearHighlightLineRange() {
   };
 }
 
-export function openConditionalPanel(line: ?number) {
+export function openConditionalPanel(line: ?number, column: ?number) {
+  console.log("openConditionalPanel", line, column);
+
   if (!line) {
     return;
   }
 
   return {
     type: "OPEN_CONDITIONAL_PANEL",
-    line
+    line,
+    column
```

Comment:
maybe we should save this as a location?

`location: {line, column, sourceId}`

The benefit of a sourceId is that 

a) it is a real location
b) in the future you can change sources it will go away and then come back and it will be there...